app/components/ui/Navigation.module.scss

```diff
@@ -75,35 +75,6 @@
   @include navLogoShared;
 }
 
-.navSearch {
-  display: inline-block;
-
-  .searchField {
-    $font-size: calc-em(15px);
-    background-repeat: no-repeat;
-    background-image: url(../../assets/img/ic-search.png);
-    background-size: 18px 18px;
-    background-position: calc-em(15px) center;
-    font-size: $font-size;
-    padding-left: calc-em(45px);
-    width: calc-em(400px);
-    height: calc-em(40px);
-    border: 0;
-    background-color: $color-grey1;
-
-    &::placeholder {
-      font-size: $font-size;
-      font-style: normal;
-    }
-  }
-}
-
-.navSearchFull {
-  @include r_max($break-tablet-p) {
-    display: none !important; // !important is needed to override .form-row
-  }
-}
-
```

Comment:
Moved to https://github.com/Exygy/askdarcel-web/pull/78/files#diff-96dd878b86ae93a22de2e7ee533339803500d6bd63fb468fde4978a08058a648R1